modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -176,40 +188,75 @@ class SegmentEdit extends Component {
 		);
 	};
 
+	_renderLocalizedInputs = () => {
+		const {
+			defaultLanguageId,
+			portletNamespace,
+			values
+		} = this.props;
+
+		const langs = Object.entries(values.name);
+
+		return (
+			langs.map(
+				([key, value]) => {
+					let render;
```

Comment:
Lets rename this to `returnVal` instead. We should keep `render` reserved, otherwise this might get a bit confusing for people reading this code later on down the road.